src/gateway/routers/axelar/Router.sol

```diff
@@ -57,9 +59,11 @@ contract AxelarRouter is Auth {
     }
 
     // --- Administration ---
-    function file(bytes32 what, address data) external auth {
+    function file(bytes32 what, bytes memory data) external auth {
```

Comment:
you can overload the file function and support both cases `address` and `bytes memory`. That way you can safe the `BytesLib` import